### PR TITLE
Add entries from the bitwarden equivalent domains list

### PR DESCRIPTION
**Pull request by superusercode**
### Checklist
- [X] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [X] The top-level JSON objects are sorted alphabetically 
- [X] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for websites-with-shared-credential-backends.json
- [X] There's evidence the domains are related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
- [X] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)

All of these shared credential sites are sourced from Bitwarden's own internal list: https://github.com/bitwarden/server/blob/master/src/Core/Utilities/StaticStore.cs

I feel strongly that all of these additions and modifications are of quality since they are a from an open source project that has hundreds of thousands of users on their own hosted offering (and many more users hosting their own instance of the service). If any of these were wrong I believe that users would have immediately submitted pull requests to modify these.

### CODE REVIEW

**Comment by superusercode** [comment]
If this PR needs to be split I understand but this will require 20+ PRs considering all the different domains they include.

**Comment by superusercode** [comment]
Additionally, lastpass equivalent domains are as follows:
```
ameritrade.com tdameritrade.com
	 
bankofamerica.com bofa.com mbna.com
	 
youtube.com google.com gmail.com
	 
apple.com icloud.com
	 
wellsfargo.com wf.com
	 
mymerrill.com ml.com merrilledge.com
	 
accountonline.com citi.com citibank.com citicards.com citibankonline.com
	 
cnet.com cnettv.com download.com news.com upload.com
	 
bananarepublic.com gap.com oldnavy.com
	 
bing.com hotmail.com live.com microsoft.com msn.com windows.com windowsazure.com office.com skype.com azure.com
	 
ua2go.com united.com unitedwifi.com
	 
overture.com yahoo.com flickr.com
	 
zonealarm.com zonelabs.com
	 
avon.com youravon.com
	 
1800contacts.com 800contacts.com
	 
amazon.com amazon.co.uk amazon.ca amazon.de amazon.fr amazon.es amazon.it amazon.com.au
	 
express-scripts.com medcohealth.com
	 
cox.com cox.net coxbusiness.com
	 
mynortonaccount.com norton.com
	 
verizon.com verizon.net
	 
logmein.com logme.in
	 
rakuten.com buy.com
	 
siriusxm.com sirius.com
	 
ea.com origin.com
	 
37signals.com basecamp.com basecamphq.com highrisehq.com
	 
steampowered.com steamcommunity.com
	 
chart.io chartio.com
	 
gotomeeting.com citrixonline.com logmeininc.com
	 
gogoair.com gogoinflight.com
	 
mysql.com oracle.com
	 
discover.com discovercard.com
	 
dcu.org dcu-online.org
	 
nefcuonline.com nefcu.com
	 
century21.com 21online.com
	 
comcast.com comcast.net xfinity.com
	 
cricketwireless.com aiowireless.com
	 
mandtbank.com mtb.com
	 
alibaba.com aliexpress.com
	 
mercadolivre.com mercadolivre.com.br mercadolibre.com
	 
taobao.com tmall.com alimama.com 1688.com
	 
zendesk.com zopim.com
	 
telekom.com t-online.de
	 
autodesk.com tinkercad.com
	 
railnation.ru railnation.de rail-nation.com railnation.gr railnation.us trucknation.de traviangames.com
	 
wpcu.coop wpcuonline.com
	 
mathletics.com mathletics.com.au mathletics.co.uk
	 
mi.com xiaomi.com
	 
facebook.com messenger.com
	 
disneymoviesanywhere.com go.com disney.com espn.com disneyplus.com d23.com shopdisney.com
	 
myuv.com uvvu.com
	 
bank-yahav.co.il bankhapoalim.co.il
	 
mdsol.com imedidata.com
	 
volvooceanrace.com virtualregatta.com
	 
mycanal.fr canal-plus.com
	 
trsretire.com divinvest.com
	 
norsk-tipping.no buypass.no
	 
my-bookings.org my-bookings.cc
	 
skygo.co.nz skytv.co.nz
	 
xiami.com alipay.com
	 
bancomer.com bancomer.com.mx bbvanet.com.mx
	 
turbotax.com intuit.com
	 
shopify.com myshopify.com
	 
concur.com concursolutions.com
	 
ebay.com ebay.at ebay.be ebay.ca ebay.ch ebay.cn ebay.co.jp ebay.co.th ebay.co.uk ebay.com.au ebay.com.hk ebay.com.my ebay.com.sg ebay.com.tw ebay.de ebay.es ebay.ie ebay.fr ebay.in ebay.it ebay.nl ebay.ph ebay.pl ebay.se ebay.vn gittigidiyor.com gmarket.co.kr
	 
schwab.com schwabplan.com
	 
hvfcu.org hvfcuonline.org
	 
firefox.com mozilla.org
	 
morganstanley.com morganstanleyclientserv.com stockplanconnect.com ms.com
	 
askubuntu.com mathoverflow.net serverfault.com stackapps.com stackexchange.com stackoverflow.com superuser.com
```

**Comment by rmondello** [comment]
I am working on making sure it’s OK for us to merge this.

**Comment by therealglazou** [comment]
That list from BitWarden is GPLv3 because extracted from a GPLv3 file... IMHO, it just cannot be integrated as is in a MIT-licensed repo w/o forcing the whole repo to become GPLv3.

**Comment by igor-makarov** [comment]
@therealglazou You raise an interesting point. However, I believe a mere list of domains cannot be considered copyrightable material.  

[This page by the US Copyright Office](https://www.copyright.gov/help/faq/faq-protect.html) has several examples of non-copyrightable materials, including specifically domain names and more generally, "facts", which I believe this list contains.  
Additionally, [this page](https://www.copyright.gov/circs/circ01.pdf) from the same site says that "[m]ere listings of ingredients or contents" are not copyrightable.

In short, I think that this list can be copied without any copyright liability.  
Nevertheless, I do think we need formal lawyer consultation. @rmondello -?

**Comment by therealglazou** [comment]
Domain names are non-copyrightable. A list of data, resulting from a work of collection, happening to be domain names, is...

**Comment by igor-makarov** [comment]
I think it's better to leave this to lawyers. I think Apple has some. 🙃

**Comment by superusercode** [comment]
I will do this one by one, forget I asked.

**Comment by rmondello** [comment]
To help navigate issues like this in the future, we’ve adopted a convention of having contributors agree to a [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md) for contributions to the project.
